setup.py

```diff
@@ -102,5 +102,6 @@
     ),
     package_data={
         "": ["*.j2"],
+        "kcidb": ["issue_editor/*", "issue_editor/templates/*"],
```

Comment:
This is a bit involved, and considering we don't actually have templates, could you just add `"*.html"` to the list of package data patterns?

Reply:
I guess  Helen may have added the `templates` directory for future use cases.
@helen-fornazier Could you please confirm if that's not the case and I can delete it?

Reply:
@spbnick Looks like a Flask application directory structure should have `templates` directory for HTML files. And that's the default location for `render_template` function (but yes, we can change that behavior).
As per the convention, we should keep it.
See https://flask.palletsprojects.com/en/2.3.x/tutorial/layout/

Reply:
OK. Then could we rename `index.html` to `index.html.j2`? Because that's what it essentially is, even if you're not using any Jinja2 there (it's still being interpreted as such). This will also make sure that the file is included into the package.

Reply:
I have renamed the template.
But I need to make changes in `setup.py` to include jinja2 templates recursively.